kbpagesd/main.go

```diff
@@ -36,6 +38,8 @@ func init() {
 		"stathat EZ key for reporting stats to stathat; empty disables stathat")
 	flag.StringVar(&fStathatPrefix, "stathat-prefix", "kbp -",
 		"prefix to stathat statnames")
+	flag.StringVar(&fBlacklist, "blacklist", "",
+		"a comma-separated list of domains to block")
```

Comment:
Not for this PR, but once we scale horizontally, then we'll need a way to make this blacklist scalable and modifiable in-place.

Reply:
Good point!

Reply:
I'll add a TODO